assets/content-popup/app/store.js

```diff
@@ -10,7 +10,7 @@ Vue.use(Vuex)
 
 function dispatchWidgetData (apiJsonData) {
   return apiJsonData.data.map( (rawWidget) => {
-    return {
+    var item = {
```

Comment:
in es6 it's better to use `const` (https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75)